examples/gnrc_minimal_xbee/Makefile

```diff
@@ -0,0 +1,44 @@
+# name of your application
+APPLICATION = gnrc_minimal_xbee
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= arduino-due
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := chronos msb-430 msb-430h
+
+# Add the XBee module and nomac
+USEMODULE += xbee
+USEMODULE += gnrc_nomac
+
```

Comment:
That's why I suggested to update the README, too. Something along the lines "to use this example with an xbee module add `USEMODULE += bla` to the `Makefile` and adapt the xbee_params.h to your local configuration"